README.md

```
@@ -1,471 +1,147 @@
-# HYXI
-![HYXI Logo](https://raw.githubusercontent.com/Veldkornet/ha-hyxi-cloud/main/custom_components/hyxi_cloud/brand/logo.png)
+![HYXI Integration for Home Assistant](https://raw.githubusercontent.com/Veldkornet/ha-hyxi-cloud/main/assets/readme-header.png)
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)